builtins/en/steps/fix.yaml

```
@@ -1,19 +1,14 @@
 name: fix
-tags: [coding]
+tags: [ coding ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)